### DISCUSSION TITLE
Plugin UI Automation not working on Cubase 11, Windows 10

### DISCUSSION THREAD

**Discussion by stuart-beattie** [apohl79/audiogridder#733] [Help]
Looking at the change log, I believe recording automation via the plugin UI was added in 1.2 beta 3. I have tried the latest beta 5 (and beta 3) but recording automation via the plugin UI just doesn't seem to work. Is this a known issue? It works fine using the generic editor. Would be amazing to have this working.

Thanks,
Stuart

**Comment by apohl79** [comment]
Created an issue, needs further investigation. Thx for reporting.
